.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+cmd/cagent/cagent
```

Comment:
1) Why is that file ignored? Never had this artefact I think
2) No new-line at the end

Reply:
That happens when need build and check without creating run configuration.
can quickly type some like CAGENT_HUB_URL="http://url" ./cagent -c new.conf

Reply:
I see.
I usually use `go run` in these cases.
`CAGENT_HUB_URL="http://url" go run main.go -c new.conf`